npm/npm_import.bzl

```diff
@@ -302,6 +303,9 @@ def npm_translate_lock(
 
             See https://github.com/bazelbuild/bazel/issues/8106
 
+        verify_patches: Verify that all patches in this folder appear in the `patches` attribute. Assumes
```

Comment:
nit: "Verify that all patches in the folder appear in the `patches` attribute. Does not recurse into subfolders."

Reply:
👍